### ISSUE TITLE
Use 'watch' on multiple parallel running tasks

### ISSUE THREAD

**Issue by dave52** [google/wireit#434]
This is perhaps a question or a feature request :)

Is it possible with Wireit, to kick off 2 (or more) tasks running in 'watch' mode in parallel, simultaneously awaiting changes?

I'm currently reaching for an `npm-run-all` (https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-p.md) package to achieve this:

```json
# package.json
...
"scripts": {
    "build": "wireit",
    "build:watch": "npm run build --watch",
    "serve": "wireit",
    "serve:watch": "npm-run-all --parallel serve build:watch",
    "test": "wireit"
  },
  "wireit": {
    "build": {
      "command": "tsc",
      "files": [
        "src/*.ts",
        "tsconfig.json"
      ],
      "output": [
        "index.js{,.map}",
        "content-smart-content.js{,.map}"
      ]
    },
    "serve": {
      "command": "web-dev-server -- --node-resolve --watch --preserve-sym-links --open demo/",
      "#comment": "TODO(aomarks) Depend only on build:ts, but needs export conditions",
      "dependencies": [
        "build"
      ]
    },
    "test": {
      "command": "wtr test/**/*.test.js --node-resolve",
      "dependencies": [
        "build"
      ]
    }
  },
...
```

I tried a couple different ways of attacking this with Wireit but came up short, let me know if it is possible and I just misconfigured a setup. It'd be amazing if Wireit could handle this, just to keep project tidy.

Let me know if you need any more info and thank you if you have any insights!

**Comment by dave52** [comment]
I think my issue is a duplicate of #33 ...going to close this
